docs/source/python_api/mlflow.entities.rst

```diff
@@ -0,0 +1,8 @@
+mlflow.entities
```

Comment:
We should probably also update docs/source/tracking.rst to describe the two APIs, right?

Reply:
Documentation is important, and I would like to get that closely reviewed. Let me do that in a follow-up PR which I'll go ahead and draft.